misc/python/materialize/test_analytics/setup/views/02-build-job.sql

```diff
@@ -112,16 +114,13 @@ INNER JOIN build_job pred_bj
 ON pred_b.build_id = pred_bj.build_id
 AND d.build_step_key = pred_bj.build_step_key
 WHERE d.predecessor_index <> 0
-AND d.branch = 'main'
 ;
 
--- only for backwards compatibility
```

Comment:
It is not a big deal to break backwards compatibility in this case. If anything goes wrong when fetching the result of the most recent builds, the former logic to fetch data from directly from Buildkite will be used as fallback.